src/main/cpp/DragonVision/DragonLimelight.cpp

```diff
@@ -217,20 +218,16 @@ units::angle::degree_t DragonLimelight::GetTargetHorizontalOffset() const
 
 units::angle::degree_t DragonLimelight::GetTargetHorizontalOffsetRobotFrame(units::length::inch_t *targetDistOffset_RF, units::length::inch_t *targetDistfromRobot_RF) const
 {
-    // Get the horizontal angle to the target and conver to radians
+    // Get the horizontal angle to the target and convert to radians
     units::angle::degree_t limelightFrameHorizAngle = GetTargetHorizontalOffset();
```

Comment:
this line seems unnecessary. Can't you just assign it to the radians variable right away?

Reply:
Done, thanks